.gitignore

```diff
@@ -1,2 +1,4 @@
-/node_modules
-/lib
+node_modules
```

Comment:
Curious as to why you prefer no leading prefix for all folders?  Note that without the leading prefix `/src/lib` would be ignored (however node_modules probably works better without them, you WOULD want to ignore `/src/node_modules` I think).

Doesn't really matter in this case - they both work the same here.

Reply:
It doesn't really bother me